index.php

```diff
@@ -0,0 +1,19 @@
+<?php declare(strict_types=1);
+require 'vendor/autoload.php';
```

Comment:
To include a third-party code library in our code. Such as twig is a code that lives [here](https://github.com/twigphp/Twig). 
Composer generates autoload file for us, to do single include in the begging of the files. Otherwise, we will need to make require/include for each file, and it will look terrible ten lines of include. 
Try to remove the `require vendor/autoload.php` temporarily and see the log what kind of error you get.